src/bson.ts

```diff
@@ -55,6 +55,9 @@ export interface BSONSerializeOptions
     > {
   /** Return BSON filled buffers from operations */
   raw?: boolean;
+
+  /** Enable utf8 validation when deserializing BSON documents.  Defaults to true. */
+  enableUtf8Validation?: boolean;
```

Comment:
Leaving this comment on the first file for visibility: let's wait to merge this PR until we decide whether or not we're going to do another patch release before the holidays